org.uqbar.project.wollok.product.ui/src/org/uqbar/project/wollok/product/ui/startup/RemoveWizardsStartup.xtend

```diff
@@ -42,7 +42,7 @@ class RemoveWizardsStartup implements WollokUIStartup {
 		val categories = registry.rootCategory.categories
 		for (wizard : categories.allWizards) {
 	        if (!wizard.id.includeWizards) {
-//	        	println("Removing wizard " + wizard.id)
+	        	//println("Removing wizard " + wizard.id)
```

Comment:
It is a minor change, but remove the commented lines, they are in the git repo if we need it back (I think this println won't be needed).